include/wlc/wlc.h

```diff
@@ -280,6 +280,12 @@ bool wlc_output_get_sleep(wlc_handle output);
 /** Wake up / sleep. */
 void wlc_output_set_sleep(wlc_handle output, bool sleep);
 
+/** Set gamma */
+void wlc_output_set_gamma(wlc_handle output, uint16_t size, uint16_t *r, uint16_t *g, uint16_t *b);
+
+/** Get gamma size */
+uint32_t wlc_output_get_gamma_size(wlc_handle output);
```

Comment:
Is this same as size given in setter above? Why is it uint32_t here and uint16_t there? Same thing for the documentation here.